pkg/alpine/Dockerfile

```diff
@@ -35,3 +35,5 @@ COPY --from=cache /mirror /mirror/
 COPY eve-alpine-deploy.sh go-compile.sh /bin/
 
 RUN apk update && apk upgrade -a
+# FIXME: workaround for riscv64 package layout
+RUN [ "$(uname -m)" != riscv64 ] || ln -s python3.9 /usr/bin/python3
```

Comment:
What is the characteristic of this error? Rather than checking arch, can we create the link based on capabilities? E.g. if python3 doesn't exist and python3.9 does (or similar), then add it?

E.g.

```Dockerfile
RUN [ -e /usr/bin/python3 ] || ln -s python3.9 /usr/bin/python3
```

Although python3.9 is kind of hard-coding as well, but this probably only is one installed, so maybe:

```Dockerfile
RUN if [ ! -e /usr/bin/python3 -a -n "$(ls python3.*)" ]; then ln -s $(ls python3.* | head -1)  /usr/bin/python3; fi
```

or similar

Reply:
Sure -- the 2nd option works for me -- let me update the PR

Reply:
Actually, once I had a minute to thing about it, we can't do any of that since the python packages itself is NOT installed at that point in the build container -- so we need to supply this link as a dangling one until the consumers of the package actually install python3.